qiita_db/util.py

```diff
@@ -697,26 +697,36 @@ def purge_filepaths(conn_handler=None):
     """
     conn_handler = conn_handler if conn_handler else SQLConnectionHandler()
 
+    # Get all the (table, column) pairs that reference to the filepath table
+    # Code adapted from http://stackoverflow.com/questions/5347050/
+    # sql-to-list-all-the-tables-that-reference-a-particular-column-in-a-table
+    table_cols_pairs = conn_handler.execute_fetchall(
+        """SELECT R.TABLE_NAME, R.column_name
```

Comment:
...at first i was afraid then I was petrified... 

This is pretty cool, good find! Following on the heels of one of @ElDeveloper prior comments on a different PR, would it be possible to cite the permanent URL for the SO post?